# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/bug1576154.sjs
# Commit: b72d1d6131cc
# Full Hash: b72d1d6131cce6845af6ffd6f83de7f993930961
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-26 09:42:55
# Description:
#   Bug 1576154 - Check mSVGDocumentWrapper when trying to report use counters; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43256
# ==============================================================================

diff -r 929421783c5b -r b72d1d6131cc dom/base/test/bug1576154.sjs
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/test/bug1576154.sjs	Sun Aug 25 22:45:33 2019 +0000
@@ -0,0 +1,8 @@
+function handleRequest(request, response)
+{
+  response.setStatusLine("1.1", 500, "Internal Server Error");
+  response.setHeader("Content-Type", "image/svg+xml", false);
+
+  let body = "<svg xmlns='http://www.w3.org/2000/svg' width='70' height='0'></svg>";
+  response.bodyOutputStream.write(body, body.length);
+}